src/Rulesets/Project/UpdateProjectRuleset.php

```diff
@@ -23,6 +24,10 @@ public function rules(): array
                 'integer',
                 new ModelExists(Project::class),
             ],
+            'client_id' => [
```

Comment:
sure about assigning projects to another client? My gut says no client_id in update.